webapp/src/controllers/app.ts

```diff
@@ -444,9 +448,13 @@ export async function editAppApi(req, res, next) {
 			if (!chatAgentModel) {
 				return dynamicResponse(req, res, 400, { error: 'Agent model invalid or missing' });
 			}
-			if (![ModelType.OPENAI, ModelType.ANTHROPIC].includes(chatAgentModel?.type)) {
+			if (
+				![ModelType.OPENAI, ModelType.ANTHROPIC, ModelType.GOOGLE_VERTEX].includes(
```

Comment:
Oops, thanks for catching this. And thanks for the better error message :slightly_smiling_face: